Remove non-standard `writing-mode: horizontal-bt`

nt1m opened the pull request:
#### ad6e194fb47b69d7bdf351c431f631e3f2908bc4
<pre>
Remove non-standard `writing-mode: horizontal-bt`
<a href="https://bugs.webkit.org/show_bug.cgi?id=244668">https://bugs.webkit.org/show_bug.cgi?id=244668</a>

Reviewed by NOBODY (OOPS!).

It&apos;s not supported by IFC or other browsers. Remove all related tests and related code.

* Source/WebCore/css/CSSPrimitiveValueMappings.h:
(WebCore::CSSPrimitiveValue::CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::operator WritingMode const):
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/css/CSSValueKeywords.in:
* Source/WebCore/css/parser/CSSParserFastPaths.cpp:
(WebCore::CSSParserFastPaths::isValidKeywordPropertyAndValue):
* Source/WebCore/dom/Element.cpp:
(WebCore::toScrollAlignmentForInlineDirection):
(WebCore::toScrollAlignmentForBlockDirection):
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.cpp:
(WebCore::LayoutIntegration::printReason):
(WebCore::LayoutIntegration::canUseForStyle):
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.h:
* Source/WebCore/platform/ScrollableArea.h:
* Source/WebCore/platform/text/WritingMode.h:
* Source/WebCore/rendering/LegacyInlineFlowBox.cpp:
(WebCore::LegacyInlineFlowBox::addBorderOutsetVisualOverflow):
* Source/WebCore/rendering/LegacyInlineFlowBox.h:
* Source/WebCore/rendering/RenderBlock.cpp:
(WebCore::RenderBlock::borderBottom const):
* Source/WebCore/rendering/RenderBox.cpp:
(WebCore::shouldFlipBeforeAfterMargins):
(WebCore::RenderBox::visualOverflowRectForPropagation const):
(WebCore::RenderBox::layoutOverflowRectForPropagation const):
(WebCore::RenderBox::flippedClientBoxRect const):
* Source/WebCore/rendering/RenderBox.h:
* Source/WebCore/rendering/RenderDetailsMarker.cpp:
(WebCore::RenderDetailsMarker::orientation const):
* Source/WebCore/rendering/RenderFlexibleBox.cpp:
(WebCore::RenderFlexibleBox::transformedWritingMode const):
(WebCore::RenderFlexibleBox::flowAwareBorderBefore const):
(WebCore::RenderFlexibleBox::flowAwareBorderAfter const):
(WebCore::RenderFlexibleBox::flowAwarePaddingBefore const):
(WebCore::RenderFlexibleBox::flowAwarePaddingAfter const):
(WebCore::RenderFlexibleBox::flowAwareMarginBeforeForChild const):
* Source/WebCore/rendering/RenderTreeAsText.cpp:
(WebCore::RenderTreeAsText::writeRenderObject):
* Source/WebCore/rendering/shapes/ShapeOutsideInfo.cpp:
(WebCore::borderBeforeInWritingMode):
(WebCore::borderAndPaddingBeforeInWritingMode):
* Source/WebCore/rendering/style/RenderStyle.cpp:
(WebCore::RenderStyle::borderBefore const):
(WebCore::RenderStyle::borderAfter const):
(WebCore::RenderStyle::borderBeforeWidth const):
(WebCore::RenderStyle::borderAfterWidth const):
* Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js:
* Tools/TestWebKitAPI/Tests/WebCore/WritingModeTests.cpp:
* LayoutTests/compositing/bounds-in-flipped-writing-mode-expected.txt:
* LayoutTests/compositing/bounds-in-flipped-writing-mode.html:
* LayoutTests/compositing/geometry/flipped-blocks-inline-mapping.html:
* LayoutTests/fast/block/dynamic-padding-border-expected.txt:
* LayoutTests/fast/block/dynamic-padding-border.html:
* LayoutTests/fast/block/float/crash-when-floating-object-is-removed.xhtml:
* LayoutTests/fast/block/float/intruding-float-not-removed-writing-mode.xhtml:
* LayoutTests/fast/block/margins-perpendicular-containing-block-expected.txt:
* LayoutTests/fast/block/margins-perpendicular-containing-block.html:
* LayoutTests/fast/block/positioning/differing-writing-modes-replaced.html:
* LayoutTests/fast/block/positioning/differing-writing-modes.html:
* LayoutTests/fast/css-grid-layout/grid-align-baseline-vertical-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-element-border-padding-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-padding-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-element-padding-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-padding-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-ltr-expected.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-ltr.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-rtl-expected.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-rtl.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-resolution-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-item-margin-resolution.html:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-height-only-expected.txt:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-height-only.html:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-width-only-expected.txt:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-width-only.html:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-expected.txt:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-in-percent-grid-expected.txt:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-in-percent-grid.html:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track.html:
* LayoutTests/fast/css-grid-layout/place-cell-by-index-expected.txt:
* LayoutTests/fast/css-grid-layout/place-cell-by-index.html:
* LayoutTests/fast/css-grid-layout/resources/grid.css:
(.horizontalTB):
(.horizontalBT): Deleted.
* LayoutTests/fast/css/logical-property-resolution-expected.txt:
* LayoutTests/fast/css/logical-property-resolution.html:
* LayoutTests/fast/css/overflow-btt-border-after-expected.txt: Removed.
* LayoutTests/fast/css/overflow-btt-border-after.html: Removed.
* LayoutTests/fast/css/sticky/sticky-writing-mode-horizontal-bt-expected.html: Removed.
* LayoutTests/fast/css/sticky/sticky-writing-mode-horizontal-bt.html: Removed.
* LayoutTests/fast/dom/HTMLMeterElement/meter-writing-mode-expected.html:
* LayoutTests/fast/dom/HTMLMeterElement/meter-writing-mode.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-writing-mode-expected.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-writing-mode.html:
* LayoutTests/fast/dom/offset-position-writing-modes-expected.txt:
* LayoutTests/fast/dom/offset-position-writing-modes.html:
* LayoutTests/fast/forms/fieldset/fieldset-flexbox-expected.html:
* LayoutTests/fast/forms/fieldset/fieldset-flexbox.html:
* LayoutTests/fast/forms/fieldset/fieldset-writing-modes-expected.html:
* LayoutTests/fast/forms/fieldset/fieldset-writing-modes.html:
* LayoutTests/fast/html/details-mouse-click-expected.txt:
* LayoutTests/fast/html/details-mouse-click.html:
* LayoutTests/fast/html/details-writing-mode-mixed.html:
* LayoutTests/fast/html/details-writing-mode.html:
* LayoutTests/fast/multicol/block-axis-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/flipped-blocks-border-after-expected.txt:
* LayoutTests/fast/multicol/flipped-blocks-border-after.html:
* LayoutTests/fast/multicol/flipped-blocks-hit-test-expected.txt: Removed.
* LayoutTests/fast/multicol/flipped-blocks-hit-test.html: Removed.
* LayoutTests/fast/multicol/hit-test-block-axis-flipped-expected.txt: Removed.
* LayoutTests/fast/multicol/hit-test-block-axis-flipped.html: Removed.
* LayoutTests/fast/multicol/hit-test-end-of-column-with-line-height-expected.txt:
* LayoutTests/fast/multicol/hit-test-end-of-column-with-line-height.html:
* LayoutTests/fast/multicol/hit-test-gap-between-pages-flipped-expected.txt: Removed.
* LayoutTests/fast/multicol/hit-test-gap-between-pages-flipped.html: Removed.
* LayoutTests/fast/multicol/newmulticol/compare-with-old-impl/hit-test-end-of-column-with-line-height-expected.txt:
* LayoutTests/fast/multicol/newmulticol/compare-with-old-impl/hit-test-end-of-column-with-line-height.html:
* LayoutTests/fast/multicol/newmulticol/hide-box-horizontal-bt-expected.html: Removed.
* LayoutTests/fast/multicol/newmulticol/hide-box-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination-h-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/multicol/pagination-h-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination-v-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/multicol/pagination-v-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/BottomToTop-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/LeftToRight-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/RightToLeft-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/TopToBottom-bt.html: Removed.
* LayoutTests/fast/multicol/progression-reverse-expected.txt:
* LayoutTests/fast/multicol/progression-reverse-overflow-expected.html:
* LayoutTests/fast/multicol/progression-reverse-overflow.html:
* LayoutTests/fast/multicol/progression-reverse.html:
* LayoutTests/fast/multicol/resources/block-axis.css:
(div.horizontal.flipped): Deleted.
* LayoutTests/fast/repaint/horizontal-bt-overflow-child-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-child.html: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-parent-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-parent.html: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-same-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-same.html: Removed.
* LayoutTests/fast/repaint/inline-horizontal-bt-overflow-expected.txt: Removed.
* LayoutTests/fast/repaint/inline-horizontal-bt-overflow.html: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block-expected.png: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block-expected.txt: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block.html: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-table-expected.txt: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-table.html: Removed.
* LayoutTests/fast/repaint/table-writing-modes-h-expected.txt:
* LayoutTests/fast/repaint/table-writing-modes-h.html:
* LayoutTests/fast/repaint/table-writing-modes-v-expected.txt:
* LayoutTests/fast/repaint/table-writing-modes-v.html:
* LayoutTests/fast/repaint/text-emphasis-v-expected.txt:
* LayoutTests/fast/repaint/text-emphasis-v.html:
* LayoutTests/fast/ruby/position-after-expected.txt:
* LayoutTests/fast/ruby/position-after.html:
* LayoutTests/fast/scrolling/programmatic-horizontal-bt-document-scroll-expected.html: Removed.
* LayoutTests/fast/scrolling/programmatic-horizontal-bt-document-scroll.html: Removed.
* LayoutTests/fast/scrolling/vertical-scrollbar-position-expected.txt:
* LayoutTests/fast/scrolling/vertical-scrollbar-position.html:
* LayoutTests/fast/table/border-collapsing/002-vertical.html:
* LayoutTests/fast/table/border-collapsing/equal-precedence-resolution-vertical.html:
* LayoutTests/fast/table/margins-perpendicular-containing-block-expected.txt:
* LayoutTests/fast/table/margins-perpendicular-containing-block.html:
* LayoutTests/fast/text/emphasis-avoid-ruby-expected.txt:
* LayoutTests/fast/text/emphasis-avoid-ruby.html:
* LayoutTests/fast/text/emphasis-overlap-expected.txt:
* LayoutTests/fast/text/emphasis-overlap.html:
* LayoutTests/fast/text/emphasis.html:
* LayoutTests/fast/text/international/vertical-text-glyph-test.html:
* LayoutTests/fast/text/international/vertical-text-metrics-test.html:
* LayoutTests/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/writing-mode/background-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/border-image-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/borders-expected.html:
* LayoutTests/fast/writing-mode/borders.html:
* LayoutTests/fast/writing-mode/box-shadow-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text-with-spelling-marker-expected.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text-with-spelling-marker.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-box-edges-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-box-edges.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-line-edges-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-line-edges.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-scroll-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-scroll.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-inline-map-local-to-container-expected.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-inline-map-local-to-container.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-text-map-local-to-container-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-text-map-local-to-container.html: Removed.
* LayoutTests/fast/writing-mode/horizontal-bt-replaced-selection.html: Removed.
* LayoutTests/fast/writing-mode/japanese-ruby-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/margin-collapse-expected.txt:
* LayoutTests/fast/writing-mode/margin-collapse.html:
* LayoutTests/fast/writing-mode/margins-expected.html:
* LayoutTests/fast/writing-mode/margins.html:
* LayoutTests/fast/writing-mode/table-hit-test-expected.txt:
* LayoutTests/fast/writing-mode/table-hit-test.html:
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-rtl.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-rtl.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-rtl.html: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/block-axis-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination-h-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination-v-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/BottomToTop-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/LeftToRight-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/RightToLeft-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/TopToBottom-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/repaint/inline-horizontal-bt-overflow-expected.png: Removed.
* LayoutTests/platform/gtk/fast/repaint/overflow-flipped-writing-mode-block-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/block/basic/fieldset-stretch-to-legend-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/differing-writing-modes-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/differing-writing-modes-replaced-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/mac/fast/multicol/block-axis-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination-h-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination-v-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/BottomToTop-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/LeftToRight-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/RightToLeft-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/TopToBottom-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/repaint/inline-horizontal-bt-overflow-expected.png: Removed.
* LayoutTests/platform/mac/fast/table/border-collapsing/002-vertical-expected.txt:
* LayoutTests/platform/mac/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt:
* LayoutTests/platform/mac/fast/text/emphasis-expected.txt:
* LayoutTests/platform/mac/fast/text/international/vertical-text-metrics-test-expected.txt:
* LayoutTests/platform/mac/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/win/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/scrollbars/scroll-rtl-or-bt-layer.html:
(WebCore::borderAndPaddingBeforeInWritingMode):
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ad6e194fb47b69d7bdf351c431f631e3f2908bc4

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/101903 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/11050 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/34975 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/111236 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/12013 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/1965 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/94308 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/108990 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/107684 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/9195 "Found 35 new test failures: css3/flexbox/flex-align-baseline.html, css3/flexbox/flex-flow-auto-margins.html, css3/flexbox/flex-flow-border.html, css3/flexbox/flex-flow-margins.html, css3/flexbox/flex-flow-padding.html, css3/flexbox/multiline-align-self.html, fast/block/positioning/differing-writing-modes-replaced.html, fast/block/positioning/differing-writing-modes.html, fast/forms/fieldset/abs-pos-child-sizing.html, fast/forms/fieldset/fieldset-overflow-auto.html ... (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/92457 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/36993 "Found 40 new test failures: css3/flexbox/flex-align-baseline.html, css3/flexbox/flex-flow-auto-margins.html, css3/flexbox/flex-flow-border.html, css3/flexbox/flex-flow-margins.html, css3/flexbox/flex-flow-padding.html, css3/flexbox/multiline-align-self.html, fast/block/basic/fieldset-stretch-to-legend.html, fast/block/positioning/differing-writing-modes-replaced.html, fast/block/positioning/differing-writing-modes.html, fast/forms/fieldset/abs-pos-child-sizing.html ... (failure)") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/91080 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/23890 "Found 30 new test failures: css3/flexbox/flex-align-baseline.html, css3/flexbox/flex-flow-auto-margins.html, css3/flexbox/flex-flow-border.html, css3/flexbox/flex-flow-margins.html, css3/flexbox/flex-flow-padding.html, css3/flexbox/multiline-align-self.html, fast/forms/fieldset/abs-pos-child-sizing.html, fast/forms/fieldset/fieldset-overflow-auto.html, fast/forms/fieldset/fieldset-overflow-hidden.html, fast/forms/fieldset/fieldset-positioned-children.html ... (failure)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/78752 "Passed tests") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/92293 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/4634 "Built successfully") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/25371 "Found 30 new test failures: css3/flexbox/flex-align-baseline.html, css3/flexbox/flex-flow-auto-margins.html, css3/flexbox/flex-flow-border.html, css3/flexbox/flex-flow-margins.html, css3/flexbox/flex-flow-padding.html, css3/flexbox/multiline-align-self.html, fast/forms/fieldset/abs-pos-child-sizing.html, fast/forms/fieldset/fieldset-overflow-auto.html, fast/forms/fieldset/fieldset-overflow-hidden.html, fast/forms/fieldset/fieldset-positioned-children.html ... (failure)") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/88469 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/83/builds/2240 "Built successfully and passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/4722 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/1813 "Found 30 new test failures: css3/flexbox/flex-align-baseline.html, css3/flexbox/flex-flow-auto-margins.html, css3/flexbox/flex-flow-border.html, css3/flexbox/flex-flow-margins.html, css3/flexbox/flex-flow-padding.html, css3/flexbox/multiline-align-self.html, fast/forms/fieldset/abs-pos-child-sizing.html, fast/forms/fieldset/fieldset-overflow-auto.html, fast/forms/fieldset/fieldset-overflow-hidden.html, fast/forms/fieldset/fieldset-positioned-children.html ... (failure)") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/46/builds/29440 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/10799 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/44859 "Passed tests") | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/91379 "Built successfully") | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/6473 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/20434 "Passed tests") | 
| | | | | 
<!--EWS-Status-Bubble-End-->

litherum reviewed (changes requested):
As discussed internally, I don’t think we can remove this until we’re more confident we won’t break content.

nt1m committed:
Remove non-standard `writing-mode: horizontal-bt`
https://bugs.webkit.org/show_bug.cgi?id=244668

Reviewed by NOBODY (OOPS!).

It's not supported by IFC or other browsers. Remove all related tests and related code.

* Source/WebCore/css/CSSPrimitiveValueMappings.h:
(WebCore::CSSPrimitiveValue::CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::operator WritingMode const):
* Source/WebCore/css/CSSProperties.json:
* Source/WebCore/css/CSSValueKeywords.in:
* Source/WebCore/css/parser/CSSParserFastPaths.cpp:
(WebCore::CSSParserFastPaths::isValidKeywordPropertyAndValue):
* Source/WebCore/dom/Element.cpp:
(WebCore::toScrollAlignmentForInlineDirection):
(WebCore::toScrollAlignmentForBlockDirection):
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.cpp:
(WebCore::LayoutIntegration::printReason):
(WebCore::LayoutIntegration::canUseForStyle):
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.h:
* Source/WebCore/platform/ScrollableArea.h:
* Source/WebCore/platform/text/WritingMode.h:
* Source/WebCore/rendering/LegacyInlineFlowBox.cpp:
(WebCore::LegacyInlineFlowBox::addBorderOutsetVisualOverflow):
* Source/WebCore/rendering/LegacyInlineFlowBox.h:
* Source/WebCore/rendering/RenderBlock.cpp:
(WebCore::RenderBlock::borderBottom const):
* Source/WebCore/rendering/RenderBox.cpp:
(WebCore::shouldFlipBeforeAfterMargins):
(WebCore::RenderBox::visualOverflowRectForPropagation const):
(WebCore::RenderBox::layoutOverflowRectForPropagation const):
(WebCore::RenderBox::flippedClientBoxRect const):
* Source/WebCore/rendering/RenderBox.h:
* Source/WebCore/rendering/RenderDetailsMarker.cpp:
(WebCore::RenderDetailsMarker::orientation const):
* Source/WebCore/rendering/RenderFlexibleBox.cpp:
(WebCore::RenderFlexibleBox::transformedWritingMode const):
(WebCore::RenderFlexibleBox::flowAwareBorderBefore const):
(WebCore::RenderFlexibleBox::flowAwareBorderAfter const):
(WebCore::RenderFlexibleBox::flowAwarePaddingBefore const):
(WebCore::RenderFlexibleBox::flowAwarePaddingAfter const):
(WebCore::RenderFlexibleBox::flowAwareMarginBeforeForChild const):
* Source/WebCore/rendering/RenderTreeAsText.cpp:
(WebCore::RenderTreeAsText::writeRenderObject):
* Source/WebCore/rendering/shapes/ShapeOutsideInfo.cpp:
(WebCore::borderBeforeInWritingMode):
(WebCore::borderAndPaddingBeforeInWritingMode):
* Source/WebCore/rendering/style/RenderStyle.cpp:
(WebCore::RenderStyle::borderBefore const):
(WebCore::RenderStyle::borderAfter const):
(WebCore::RenderStyle::borderBeforeWidth const):
(WebCore::RenderStyle::borderAfterWidth const):
* Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js:
* Tools/TestWebKitAPI/Tests/WebCore/WritingModeTests.cpp:
* LayoutTests/compositing/bounds-in-flipped-writing-mode-expected.txt:
* LayoutTests/compositing/bounds-in-flipped-writing-mode.html:
* LayoutTests/compositing/geometry/flipped-blocks-inline-mapping.html:
* LayoutTests/fast/block/dynamic-padding-border-expected.txt:
* LayoutTests/fast/block/dynamic-padding-border.html:
* LayoutTests/fast/block/float/crash-when-floating-object-is-removed.xhtml:
* LayoutTests/fast/block/float/intruding-float-not-removed-writing-mode.xhtml:
* LayoutTests/fast/block/margins-perpendicular-containing-block-expected.txt:
* LayoutTests/fast/block/margins-perpendicular-containing-block.html:
* LayoutTests/fast/block/positioning/differing-writing-modes-replaced.html:
* LayoutTests/fast/block/positioning/differing-writing-modes.html:
* LayoutTests/fast/css-grid-layout/grid-align-baseline-vertical-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-element-border-padding-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-border-padding-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-element-padding-grid-item-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-element-padding-grid-item.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-ltr-expected.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-ltr.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-rtl-expected.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-auto-columns-rows-horizontal-rtl.html:
* LayoutTests/fast/css-grid-layout/grid-item-margin-resolution-expected.txt:
* LayoutTests/fast/css-grid-layout/grid-item-margin-resolution.html:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-height-only-expected.txt:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-height-only.html:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-width-only-expected.txt:
* LayoutTests/fast/css-grid-layout/minmax-fixed-logical-width-only.html:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-expected.txt:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-in-percent-grid-expected.txt:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track-in-percent-grid.html:
* LayoutTests/fast/css-grid-layout/percent-grid-item-in-percent-grid-track.html:
* LayoutTests/fast/css-grid-layout/place-cell-by-index-expected.txt:
* LayoutTests/fast/css-grid-layout/place-cell-by-index.html:
* LayoutTests/fast/css-grid-layout/resources/grid.css:
(.horizontalTB):
(.horizontalBT): Deleted.
* LayoutTests/fast/css/logical-property-resolution-expected.txt:
* LayoutTests/fast/css/logical-property-resolution.html:
* LayoutTests/fast/css/overflow-btt-border-after-expected.txt: Removed.
* LayoutTests/fast/css/overflow-btt-border-after.html: Removed.
* LayoutTests/fast/css/sticky/sticky-writing-mode-horizontal-bt-expected.html: Removed.
* LayoutTests/fast/css/sticky/sticky-writing-mode-horizontal-bt.html: Removed.
* LayoutTests/fast/dom/HTMLMeterElement/meter-writing-mode-expected.html:
* LayoutTests/fast/dom/HTMLMeterElement/meter-writing-mode.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-writing-mode-expected.html:
* LayoutTests/fast/dom/HTMLProgressElement/progress-writing-mode.html:
* LayoutTests/fast/dom/offset-position-writing-modes-expected.txt:
* LayoutTests/fast/dom/offset-position-writing-modes.html:
* LayoutTests/fast/forms/fieldset/fieldset-flexbox-expected.html:
* LayoutTests/fast/forms/fieldset/fieldset-flexbox.html:
* LayoutTests/fast/forms/fieldset/fieldset-writing-modes-expected.html:
* LayoutTests/fast/forms/fieldset/fieldset-writing-modes.html:
* LayoutTests/fast/html/details-mouse-click-expected.txt:
* LayoutTests/fast/html/details-mouse-click.html:
* LayoutTests/fast/html/details-writing-mode-mixed.html:
* LayoutTests/fast/html/details-writing-mode.html:
* LayoutTests/fast/multicol/block-axis-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/flipped-blocks-border-after-expected.txt:
* LayoutTests/fast/multicol/flipped-blocks-border-after.html:
* LayoutTests/fast/multicol/flipped-blocks-hit-test-expected.txt: Removed.
* LayoutTests/fast/multicol/flipped-blocks-hit-test.html: Removed.
* LayoutTests/fast/multicol/hit-test-block-axis-flipped-expected.txt: Removed.
* LayoutTests/fast/multicol/hit-test-block-axis-flipped.html: Removed.
* LayoutTests/fast/multicol/hit-test-end-of-column-with-line-height-expected.txt:
* LayoutTests/fast/multicol/hit-test-end-of-column-with-line-height.html:
* LayoutTests/fast/multicol/hit-test-gap-between-pages-flipped-expected.txt: Removed.
* LayoutTests/fast/multicol/hit-test-gap-between-pages-flipped.html: Removed.
* LayoutTests/fast/multicol/newmulticol/compare-with-old-impl/hit-test-end-of-column-with-line-height-expected.txt:
* LayoutTests/fast/multicol/newmulticol/compare-with-old-impl/hit-test-end-of-column-with-line-height.html:
* LayoutTests/fast/multicol/newmulticol/hide-box-horizontal-bt-expected.html: Removed.
* LayoutTests/fast/multicol/newmulticol/hide-box-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination-h-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/multicol/pagination-h-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination-v-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/multicol/pagination-v-horizontal-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/BottomToTop-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/LeftToRight-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/RightToLeft-bt.html: Removed.
* LayoutTests/fast/multicol/pagination/TopToBottom-bt.html: Removed.
* LayoutTests/fast/multicol/progression-reverse-expected.txt:
* LayoutTests/fast/multicol/progression-reverse-overflow-expected.html:
* LayoutTests/fast/multicol/progression-reverse-overflow.html:
* LayoutTests/fast/multicol/progression-reverse.html:
* LayoutTests/fast/multicol/resources/block-axis.css:
(div.horizontal.flipped): Deleted.
* LayoutTests/fast/repaint/horizontal-bt-overflow-child-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-child.html: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-parent-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-parent.html: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-same-expected.txt: Removed.
* LayoutTests/fast/repaint/horizontal-bt-overflow-same.html: Removed.
* LayoutTests/fast/repaint/inline-horizontal-bt-overflow-expected.txt: Removed.
* LayoutTests/fast/repaint/inline-horizontal-bt-overflow.html: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block-expected.png: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block-expected.txt: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-block.html: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-table-expected.txt: Removed.
* LayoutTests/fast/repaint/overflow-flipped-writing-mode-table.html: Removed.
* LayoutTests/fast/repaint/table-writing-modes-h-expected.txt:
* LayoutTests/fast/repaint/table-writing-modes-h.html:
* LayoutTests/fast/repaint/table-writing-modes-v-expected.txt:
* LayoutTests/fast/repaint/table-writing-modes-v.html:
* LayoutTests/fast/repaint/text-emphasis-v-expected.txt:
* LayoutTests/fast/repaint/text-emphasis-v.html:
* LayoutTests/fast/ruby/position-after-expected.txt:
* LayoutTests/fast/ruby/position-after.html:
* LayoutTests/fast/scrolling/programmatic-horizontal-bt-document-scroll-expected.html: Removed.
* LayoutTests/fast/scrolling/programmatic-horizontal-bt-document-scroll.html: Removed.
* LayoutTests/fast/scrolling/vertical-scrollbar-position-expected.txt:
* LayoutTests/fast/scrolling/vertical-scrollbar-position.html:
* LayoutTests/fast/table/border-collapsing/002-vertical.html:
* LayoutTests/fast/table/border-collapsing/equal-precedence-resolution-vertical.html:
* LayoutTests/fast/table/margins-perpendicular-containing-block-expected.txt:
* LayoutTests/fast/table/margins-perpendicular-containing-block.html:
* LayoutTests/fast/text/emphasis-avoid-ruby-expected.txt:
* LayoutTests/fast/text/emphasis-avoid-ruby.html:
* LayoutTests/fast/text/emphasis-overlap-expected.txt:
* LayoutTests/fast/text/emphasis-overlap.html:
* LayoutTests/fast/text/emphasis.html:
* LayoutTests/fast/text/international/vertical-text-glyph-test.html:
* LayoutTests/fast/text/international/vertical-text-metrics-test.html:
* LayoutTests/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/fast/writing-mode/background-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/border-image-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/borders-expected.html:
* LayoutTests/fast/writing-mode/borders.html:
* LayoutTests/fast/writing-mode/box-shadow-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text-with-spelling-marker-expected.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text-with-spelling-marker.html: Removed.
* LayoutTests/fast/writing-mode/english-bt-text.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-box-edges-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-box-edges.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-line-edges-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-line-edges.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-scroll-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow-scroll.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-hit-test-overflow.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-inline-map-local-to-container-expected.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-inline-map-local-to-container.html: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-text-map-local-to-container-expected.txt: Removed.
* LayoutTests/fast/writing-mode/flipped-blocks-text-map-local-to-container.html: Removed.
* LayoutTests/fast/writing-mode/horizontal-bt-replaced-selection.html: Removed.
* LayoutTests/fast/writing-mode/japanese-ruby-horizontal-bt.html: Removed.
* LayoutTests/fast/writing-mode/margin-collapse-expected.txt:
* LayoutTests/fast/writing-mode/margin-collapse.html:
* LayoutTests/fast/writing-mode/margins-expected.html:
* LayoutTests/fast/writing-mode/margins.html:
* LayoutTests/fast/writing-mode/table-hit-test-expected.txt:
* LayoutTests/fast/writing-mode/table-hit-test.html:
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/div-x-horizontal-bt-rtl.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-x-horizontal-bt-rtl.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-ltr-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-ltr.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-rtl-expected.html: Removed.
* LayoutTests/imported/blink/fast/pagination/viewport-y-horizontal-bt-rtl.html: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/block-axis-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination-h-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination-v-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/BottomToTop-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/LeftToRight-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/RightToLeft-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/multicol/pagination/TopToBottom-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/repaint/inline-horizontal-bt-overflow-expected.png: Removed.
* LayoutTests/platform/gtk/fast/repaint/overflow-flipped-writing-mode-block-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/gtk/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/ios/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/ios/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/block/basic/fieldset-stretch-to-legend-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/differing-writing-modes-expected.txt:
* LayoutTests/platform/mac/fast/block/positioning/differing-writing-modes-replaced-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/mac/fast/multicol/block-axis-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination-h-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination-v-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/BottomToTop-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/LeftToRight-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/RightToLeft-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/TopToBottom-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/repaint/inline-horizontal-bt-overflow-expected.png: Removed.
* LayoutTests/platform/mac/fast/table/border-collapsing/002-vertical-expected.txt:
* LayoutTests/platform/mac/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt:
* LayoutTests/platform/mac/fast/text/emphasis-expected.txt:
* LayoutTests/platform/mac/fast/text/international/vertical-text-metrics-test-expected.txt:
* LayoutTests/platform/mac/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/background-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/border-image-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/box-shadow-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/english-bt-text-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/horizontal-bt-replaced-selection-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/mac/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/mac/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/win/fast/writing-mode/japanese-ruby-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/win/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/BottomToTop-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/LeftToRight-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/RightToLeft-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/multicol/pagination/TopToBottom-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/background-horizontal-bt-expected.png: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/border-image-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/box-shadow-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/multicol/block-axis-horizontal-bt-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/english-bt-text-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/horizontal-bt-replaced-selection-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/writing-mode/japanese-ruby-horizontal-bt-expected.txt: Removed.
* LayoutTests/scrollbars/scroll-rtl-or-bt-layer.html:
(WebCore::borderAndPaddingBeforeInWritingMode):
